src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Context/Migrations/201804061714348_initial.cs

```diff
@@ -0,0 +1,18 @@
+namespace VolleyManagement.Data.MsSql.Context.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class initial : DbMigration
```

Comment:
Name?

Reply:
is it good Location? for a name this class?